rune/libenclave/internal/runtime/pal/api_linux_v3.go

```diff
@@ -16,12 +34,43 @@ import "C"
 import (
 	"fmt"
 	"github.com/inclavare-containers/rune/libenclave/intelsgx"
+	"github.com/sirupsen/logrus"
 	"unsafe"
 )
 
 type enclaveRuntimePalApiV3 struct {
 }
 
+func (pal *enclaveRuntimePalApiV3) get_version() uint32 {
```

Comment:
You don't need to re-implement this function in v3.

Reply:
You Can refer to API v2 which doesn't re-implement this API.